Makefile.vita

```diff
@@ -1,7 +1,7 @@
 NAME    := noods
 BUILD   := build-vita
 SOURCES := src src/common src/vita
-ARGS    := -Ofast -flto -std=c++11 #-DDEBUG
+ARGS    := -Ofast -flto -std=c++11 -march=armv7-A -mtune=generic-armv7-a #-DDEBUG
```

Comment:
Minor thing, but it should be `armv7-a` instead of `armv7-A`. Not sure if it actually matters but I'd rather keep it consistent 🙂